src/metabase/models/setting.clj

```diff
@@ -1293,8 +1245,8 @@
                (when api/*is-superuser?*
                  [:admin]))))
 
-(defn- filtered-user-facing-settings
-  "Docstrings are hard"
+(defn- user-facing-settings-matching
+  "Returns the user facing view of the registered settings satisfying the given predicate"
```

Comment:
@piranha here's the rename i went with based on your nit in the previous PR

Reply:
Nice!